Sources/TuistCore/Models/XCFrameworkInfoPlist.swift

```diff
@@ -0,0 +1,42 @@
+import Basic
+import Foundation
+
+public struct XCFrameworkInfoPlist: Codable {
```

Comment:
What about adding an integration test using a real .xcframework?

Reply:
yes, i'll get it integrated too